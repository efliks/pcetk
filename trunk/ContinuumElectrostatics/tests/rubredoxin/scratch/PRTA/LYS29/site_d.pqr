ATOM      0  CE  LYS    29      18.485  25.711   3.510   0.10000   1.700      0      0      0      0
ATOM      0  HE1 LYS    29      18.676  25.400   4.550   0.00000   1.000      0      0      0      0
ATOM      0  HE2 LYS    29      19.210  26.510   3.255   0.00000   1.000      0      0      0      0
ATOM      0  NZ  LYS    29      18.778  24.539   2.602  -0.25000   1.550      0      0      0      0
ATOM      0  HZ1 LYS    29      18.558  24.833   1.628   0.05000   0.300      0      0      0      0
ATOM      0  HZ2 LYS    29      18.145  23.768   2.899   0.05000   0.300      0      0      0      0
ATOM      0  HZ3 LYS    29      19.771  24.262   2.710   0.05000   0.300      0      0      0      0
