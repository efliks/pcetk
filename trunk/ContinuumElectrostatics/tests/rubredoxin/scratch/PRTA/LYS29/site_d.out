Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/LYS29//site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (13.319, 37.4012, 4.51304)) 121 2
Finer level:    (18.8033, 25.0034, 3.02202) 101 1
Finer level:    (18.8033, 25.0034, 3.02202) 101 0.5
Finest level:   (18.8033, 25.0034, 3.02202) 101 0.25


Self energy of scratch/PRTA/LYS29//site_d in three dielectric environment = 19.6735
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS29//site_d with background charge set in three dielectric environment = -0.177312
(probably in kcal/mole)



Interaction energies of scratch/PRTA/LYS29//site_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.00129121
0   1 : -1.07405e-05
1   0 : 3.87e-06
1   1 : -0.000549436
2   0 : 7.19554e-05
2   1 : 3.44473e-06
3   0 : -2.18021e-05
3   1 : -0.000200568
4   0 : -0.000136648
4   1 : -0.00100911
5   0 : -0.000900286
5   1 : -0.0106192
6   0 : -0.000232744
6   1 : -0.00289596
7   0 : 0.000262485
7   1 : -0.0354745
8   0 : -0.000113338
8   1 : -0.00391978
9   0 : 0.000215595
9   1 : -0.000720709
10   0 : -5.4592e-07
10   1 : -0.000242277
11   0 : 66.0222
11   1 : 39.3454
12   0 : -0.00236668
12   1 : -0.0120716
13   0 : -0.00223639
13   1 : -0.0175853
14   0 : -2.26571e-06
14   1 : -0.000164201
15   0 : -1.84253e-05
15   1 : -0.000247744
16   0 : -2.29505e-05
16   1 : -0.000574402
17   0 : -2.3873e-05
17   1 : -0.000163232
18   0 : 0.00159198
18   1 : -2.31415e-08
19   0 : 9.16877e-06
19   1 : -5.38516e-05
20   0 : 1.35528e-05
20   1 : -0.000107645
21   0 : 0.00556991
21   1 : 2.16333e-05
22   0 : -2.96091e-05
22   1 : -0.000716982
23   0 : 1.97025e-05
23   1 : -1.02604e-05
Total runtime: 15.48 s user 0.15 s system 
